tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -123,8 +119,6 @@ spec:
   secret: osp-secret
   serviceUser: heat
   transportURLSecret: rabbitmq-transport-url-heat-heat-transport
-status:
```

Comment:
I checked the other operators (eg. keystone-operator) and readyCount is asserted. Should we keep this ? also we have the same field for HeatEngine CR and can be asserted.

Reply:
added both of them